tests/e2e/TestConstants.ts

```diff
@@ -54,6 +54,24 @@ export const TestConstants = {
      */
     TS_SELENIUM_RESOLUTION_HEIGHT: Number(process.env.TS_SELENIUM_RESOLUTION_HEIGHT) || 1080,
 
+    /**
+     * Editor the tests are running against, "code" by default.
+     * Possible values: "code", "theia"
+     */
+    TS_SELENIUM_EDITOR: process.env.TS_SELENIUM_EDITOR || 'code',
+
+    /**
+     * Base version of VSCode editor for monaco-page-objects, "1.37.0" by default.
+     */
+    TS_SELENIUM_CODE_BASE_VERSION: process.env.TS_SELENIUM_CODE_BASE_VERSION || '1.37.0',
```

Comment:
So that this repo is for Che Theia editors test as well, I would suggest to name constant more specifically, like TS_SELENIUM_MONACO_PAGE_OBJECTS_VERSION